front-end/src/components/Post/GovernanceSideBar/Referenda/ReferendumVoteInfo.tsx

```diff
@@ -91,30 +95,37 @@ const ReferendumVoteInfo = ({ className, referendumId, threshold }: Props) => {
 	}, [electorate, nayVotes, threshold, turnout]);
 
 	return (
-		<Card className={className}>
-			<h3>Overview</h3>
-			<VoteProgress
-				ayeVotes={ayeVotes}
-				className='vote-progress'
-				passingThreshold={passingThreshold}
-				nayVotes={nayVotes}
-			/>
-			<Grid columns={3} divided>
-				<Grid.Row>
-					<Grid.Column>
-						<h6>Turnout</h6>
-						<div>{formatBnBalance(turnout, { numberAfterComma: 2 })}</div>
-					</Grid.Column>
-					<Grid.Column width={5}>
-						<h6>Aye</h6>
-						<div>{formatBnBalance(ayeVotes, { numberAfterComma: 2 })}</div>
-					</Grid.Column>
-					<Grid.Column width={5}>
-						<h6>Nay</h6>
-						<div>{formatBnBalance(nayVotes, { numberAfterComma: 2 })}</div>
-					</Grid.Column>
-				</Grid.Row>
-			</Grid>
+		<Card className={loadingStatus.isLoading ? `LoaderWrapper ${className}` : className}>
+			{loadingStatus.isLoading
+				?
+				<Loader text={loadingStatus.message} timeout={3000} timeoutmessage='Api is unresponsive.'/>
```

Comment:
The timeout message could / should maybe be taken from `loadingStatus.message`, but this would mean using `setLoadingStatus` again in the api checks above

Reply:
I'm not sure I understand what you mean. Sure the LoadingStatusType could have a `timeoutMessage` or something so that all info for the Loader go into this object, but it's fine like this I think.